app/assets/stylesheets/modern-business.scss

```diff
@@ -94,12 +93,20 @@ header.carousel .fill#slider-3 {
   padding-top: 60px;
 }
 
-#header-into {
+#header-intro {
   text-align: center;
   font-size: 18px;
 }
 
-/* Flash Notice Styling */
+.donation-candidate h3:hover {
+  color: #ff0000
```

Comment:
Color literals like `#ff0000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Declaration should be terminated by a semicolon